### DISCUSSION TITLE
Library generation using data from PeptideAtlas

### DISCUSSION THREAD

**Discussion by andrewlai** [vdemichev/DiaNN#1247] [General]
Hi Vadim,

First of all, I’d like to thank you for making this great software available to everyone!

I have a question about potentially using PeptideAtlas data for library generation versus the entire human FASTA.

For example, if I am working with extracellular vesicles from human plasma, is there any advantage to using their plasma_EV build (https://peptideatlas.org/builds/human/plasma_ev/)? If so, would I simply download the peptide sequences in FASTA format, generate the library, and then reannotate the predicted library using the UniProt human FASTA file?

**Comment by vdemichev** [comment]
Hi Andrew,

I am not sure about the plasma EV build, but in general restricting the set of peptides you are looking for is indeed beneficial for plasma. Further, many peptides found in plasma are non-tryptic but are present in PeptideAtlas builds. So yes, searching against PeptideAtlas is worth a try. However, with good enriched plasma data you will get also quite good results by just searching against the whole human database.

> reannotate the predicted library using the UniProt human FASTA file?

Will not work, I think there are a lot of non-canonical stuff in there, need to pull annotations from somewhere else (using R). I am not sure where this info was (looked at PeptideAtlas years ago), but I remeber managing to find these annotations for PeptideAtlas peptides somewhere.

Best,
Vadim
